dask_cuda/explicit_comms/dataframe_merge.py

```diff
@@ -228,66 +122,36 @@ def df_concat(df_parts):
         return await hash_join(len(workers), rank, eps, df1, df2, left_on, right_on)
 
 
-def dataframe_merge(left, right, on=None, left_on=None, right_on=None, how="inner"):
-    """Merge two Dask DataFrames
+def dataframe_merge(left, right, on=None, left_on=None, right_on=None):
+    """Merge two DataFrames using explicit-comms.
 
-    This will merge the two datasets, either on the indices, a certain column
-    in each dataset or the index in one dataset and the column in another.
+    This is an explicit-comms version of Dask's Dataframe.merge() that
```

Comment:
No need to address in this Pr,  but "left" is pretty common and shouldn't be difficult to support (although, I may be missing something)